client/a8c-for-agencies/sections/referrals/lib/get-estimated-commission.ts

```diff
@@ -17,62 +17,80 @@ export const getDailyPrice = ( product: APIProductFamilyProduct, quantity: numbe
 export const getEstimatedCommission = (
```

Comment:
Most changes in this file are just indention changes and moving a couple lines that are only needed for legacy calculations. The main change is branching depending on it having a BD commission or not. The legacy calculation logic should not have changed at all.